### DISCUSSION TITLE
Hi im new here and there are some questions with building...

### DISCUSSION THREAD

**Discussion by EvATive7** [PowerShell/PowerShell#24695] [Q&A] [answered]
Platform: Windows 11, x64
Code Version: master, commit 617dbda8f47cf06d115947f0db282e1994294604
.NET Version: 9.0.100

I cannot build in VS, actually when I press the build&run button
![图片](https://github.com/user-attachments/assets/19b28faa-eba6-415c-aedf-256f73f2952f)
, I got many problems seems I cannot solve it...I can successfully building with `Start-PSBuild`  
according doc and #3400 , building in VS is not supported right? May I ask how does everyone debug it easily?

**Comment by MartinGC94** [comment, marked as answer]
My development process here is to have a pwsh window open that I run `Start-PSBuild` from whenever I make changes. When it's done I launch a new instance from the run menu (Win+R) and from VS I press the attach debugger button and select the newly built instance.  
It's a little inconvenient and I'd prefer if I could simply press the "play" button from VS but it's not so bad that I want to put in the time to learn how to fix it and open a PR for it.

**Reply by EvATive7** [reply]
@MartinGC94 Thank you very much for letting me know it isn't caused by my environment! (And it saves a lot of time for me to fix it😹😹)
Yes it's inconvenient to build and test under this situation, I hope someone in this project can make it, and I'd be happy to help if I can😔
